### PR TITLE
fix flow

### PR DESCRIPTION
**Pull request by AntonPuko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,104 +1,105 @@
-{
```
Comment (AntonPuko) [inline package.json]: idk, why git thinks that I change the full file, I only installed `nullthrows`

Comment (jlkalberer) [inline package.json]: Yeah, we need to figure that out.  I think webstorm changes the line endings.

File: src/types.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ export type Event = EventData & {
 };
 
 export type EventData = {
-  data?: string,
+  data?: ?string,
```
Comment (jlkalberer) [inline src/types.js]: Do we really need this? I'd prefer either nullable or optional but not both.

Comment (AntonPuko) [inline src/types.js]: ok. i'll change postbody.data type in eventController instead

**Pull request merged**